workers/generic-worker/artifacts_multiuser.go

```diff
@@ -2,8 +2,11 @@
 
 package main
 
-import "github.com/taskcluster/taskcluster/v58/workers/generic-worker/process"
+import (
+	"github.com/taskcluster/taskcluster/v58/workers/generic-worker/process"
+	"github.com/taskcluster/taskcluster/v58/workers/generic-worker/runtime"
+)
 
-func gwCopyToTempFile(exe, filePath string) (*process.Command, error) {
-	return process.NewCommandNoOutputStreams([]string{exe, "copy-to-temp-file", "--copy-file", filePath}, taskContext.TaskDir, []string{}, taskContext.pd)
+func gwCopyToTempFile(filePath string) (*process.Command, error) {
+	return process.NewCommandNoOutputStreams([]string{runtime.GenericWorkerBinary(), "copy-to-temp-file", "--copy-file", filePath}, taskContext.TaskDir, []string{}, taskContext.pd)
```

Comment:
I notice that the multiuser version uses `NewCommandNoOutputStreams` but the simple version uses `NewCommand`. Is there a reason for the difference?